stdlib/src/collections/list.mojo

```diff
@@ -753,3 +754,30 @@ struct List[T: CollectionElement](CollectionElement, Sized, Boolable):
             The UnsafePointer to the underlying memory.
         """
         return self.data
+
+    @always_inline
+    fn __contains__[
+        T2: ComparableCollectionElement
+    ](self: List[T2], value: T) -> Bool:
+        """Verify if a given value is present in the list.
+
+        ```mojo
+        var x = List[Int](1,2,3)
+        if 3 in x: print("x contains 3")
+        ```
+        Parameters:
+            T2: The type of the elements in the list. Must implement the
+              traits `EqualityComparable` and `CollectionElement`.
+
+        Args:
+            value: The value to find.
+
+        Returns:
+            True or False.
```

Comment:
**Suggestion** Reword to "True if the value is contained in the list, False otherwise." or something of that nature.  "True or False" doesn't help the reader of API docs much for this function.  They can infer that from the nature of the return type (`Bool`).  This doc string part should tell them _when_ it returns one or the other (i.e. the semantics of the function).